src/components/PaginationV2/PaginationV2.js

```diff
@@ -139,9 +129,7 @@ export default class PaginationV2 extends Component {
   };
 
   static defaultProps = {
-    backwardText: 'Backward',
     itemRangeText: (min, max, total) => `${min}-${max} of ${total} items`,
-    forwardText: 'Forward',
```

Comment:
Shouldn't we be setting the new text as default, not as an unchangeable hardcoded value? Thinking of internationalization

Reply:
@tw15egan oh wow yes true!!!